### PR TITLE
feat(cli): `hive app:check` command

### PR DESCRIPTION
**Pull request by n1ru4l**
Adds `hive app:check` to validate persisted app operations against the target’s latest valid GraphQL schema without creating or publishing an app deployment.

The `app:check` commands extract and validates the operations the same way the schema registry is validating the documents before storing them.

**NOTE**: The existing `operations:check` command is more general purpose and already has different loading behaviour compared to how we load the persisted documents manifest (around `#import`), thus it is more straight forward to introduce this new command that shares the expected behaviour with how other app deployment related commands work.

## Example Command Usages

**Manifest File**

```
hive app:check persisted-documents.json \
  --registry.accessToken "$HIVE_TOKEN" \
  --target my-org/my-project/production
```

**Directory with GraphQL Files**

```
hive app:check ./src/operations \
  --registry.accessToken "$HIVE_TOKEN" \
  --target my-org/my-project/production
```

**Glob**

```
hive app:check "src/**/*.graphql" \
  --registry.accessToken "$HIVE_TOKEN" \
  --target my-org/my-project/production
```

## Example Command Output

```bash
> hive app:check ../../web/app/src/gql/persisted-documents.json

=== Summary
Total: 212
Invalid: 212 (100%)

=== Details
Error: Invalid operation syntax:
✖ ../../web/app/src/gql/persisted-documents.json (sha256:bc2faab99e07f170ab859f2f6a2626af7b95af74f84fb50f5b02e090b5053679)
 - Unknown type AdminOperationPoint.
 - Unknown type DateRangeInput.
 - Cannot query field admin on type Query.
✖ ../../web/app/src/gql/persisted-documents.json (sha256:cf7758fb01e287fe99182d4d8fbcf38bb004d3f11befa7869a8efb9552f87d59)
 - Unknown type OrganizationGetStarted.
 - Unknown type OrganizationConnection.
 - Unknown type Organization.
 - Unknown type Organization.
 - Unknown type User.
 - Unknown type OrganizationConnection.
 - Unknown type Organization.
 - Cannot query field me on type Query.
 - Cannot query field organizationBySlug on type Query.
 - Cannot query field organizations on type Query.
✖ ../../web/app/src/gql/persisted-documents.json (sha256:bbc76ec4af947b9cb434dd4106cf4cac40319cdc6e658ee4a0dc9379d16b83cb)
 - Unknown type CreateProjectInput.
 ```

### CODE REVIEW

File: integration-tests/tests/cli/app.spec.ts [hunk 1]
```diff
@@ -136,6 +136,69 @@ test('app:create --version is optional and auto-generates a version', async () =
   expect(output).toContain('created');
 });
 
+test('app:check validates an Apollo manifest against the latest schema', async () => {
```
Comment (jdolle) [inline integration-tests/tests/cli/app.spec.ts:139]: missing a test for if the file doesnt exist or there are zero documents

File: packages/libraries/cli/src/commands/app/check.ts [hunk 2]
```diff
@@ -0,0 +1,174 @@
...
+      if (!sdl) {
+        throw new SchemaNotFoundError();
+      }
+
+      const schema = buildSchema(sdl, {
+        assumeValidSDL: true,
+        assumeValid: true,
+      });
+
+      const invalidOperations = operations.flatMap(operation => {
```
Comment (jdolle) [inline packages/libraries/cli/src/commands/app/check.ts:130]: Should this validate in batches (e.g. using plimit) to avoid overwhelming the cpu? There can be a _lot_ of documents.

**Review by jdolle — APPROVED** [review]
Nothing concerning. Just a couple recommendations

File: packages/libraries/cli/src/commands/app/check.ts (hunk 2 above)
Comment (n1ru4l) [reply packages/libraries/cli/src/commands/app/check.ts:130]: This is all sync operations, no promises involved.

File: integration-tests/tests/cli/app.spec.ts (hunk 1 above)
Comment (n1ru4l) [reply integration-tests/tests/cli/app.spec.ts:139]: done
